src/common/motech-list/list.html

```diff
@@ -2,6 +2,7 @@
     <header ng-transclude="header"></header>
     <nav>
         <ul class="motech-list-column-headers">
+            <span class="motech-list-loading-state" ng-if="isLoading">{{'The motech-list component is loading...'|translate}}</span>
```

Comment:
The translation here should be a message token

I like the positioning of this element — I don't think it makes semantic sense to put this warning here because:
(a) a `<span>` shouldn't be the child of a `<ul>`
(b) if a screen reader was reading the page, ideally the first message on the list should be "loading" — indicating to the user that there is nothing else to pay attention to...